mcpwm_foc.c

```diff
@@ -3928,8 +3928,8 @@ static float correct_hall(float angle, float dt, volatile motor_all_state_t *mot
 	motor->m_hall_dt_diff_now += dt;
 
 	float rad_per_sec = (M_PI / 3.0) / motor->m_hall_dt_diff_last;
-	float rpm_abs_fast = fabsf(motor->m_speed_est_fast / ((2.0 * M_PI) / 60.0));
-	float rpm_abs_hall = fabsf(rad_per_sec / ((2.0 * M_PI) / 60.0));
```

Comment:
Note that this was a division by `RPM2RADPS`, so I turned it into a simple multiplication of `RADPS2RPM`.